deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessorException.java

```diff
@@ -0,0 +1,20 @@
+package com.airbnb.deeplinkdispatch;
+
+import javax.lang.model.element.Element;
+
+class DeepLinkProcessorException extends RuntimeException {
```

Comment:
this should be in kotlin. https://github.com/airbnb/DeepLinkDispatch/pull/260 is adding kotlin dependency

Reply:
done